spec/features/recipes_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.feature 'Recipes management', type: :feature do
+  let(:user) { create(:user) }
+
+  before do
+    login_as(user, scope: :user)
+    visit recipes_path
+  end
+
+  scenario 'User views recipe index page' do
+    expect(page).to have_content('Find your favourite recipe here')
+    expect(page).to have_link('Add a Recipe')
+  end
+
+  scenario 'User adds a new recipe' do
+    click_link 'Add a Recipe'
+    expect(page).to have_current_path(new_recipe_path)
+
+    expect(page).to have_content('Add a new recipe')
+  end
```

Comment:
- [ ] I tried to find the issue with this and I figured that the `Add a Recipe` that you are trying to click is not on the page when it kickstarts, and I tried to use the Add Recipe and it also failed because there are two Add Recipe on the page 
![image](https://github.com/fatmahussein/Recipe-app/assets/86200356/79ac334c-5fd4-48d7-a255-d403ed24b7b9)

I went further to rename one of it and it passed, so double check it and make the necessary corrections, please.

![image](https://github.com/fatmahussein/Recipe-app/assets/86200356/2c4c1a75-820f-40e6-bf84-952a718f9193)